heartbeat/monitors/active/icmp/stdloop.go

```diff
@@ -0,0 +1,402 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
There are very few new changes in this file, it's just been moved from `loop.go` but the old filename didn't make much sense. I'll comment where the actual changes are